docs/pages/setup/reference/resources.mdx

```diff
@@ -4,52 +4,74 @@ description: The detailed reference documentation for Teleport Configuration Res
 h1: Teleport Configuration Resources Reference
 ---
 
-## Resources
+## Introduction
+
+<Tabs>
+<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
 
 A Teleport administrator has two tools to configure a Teleport cluster:
 
-- The [configuration file](./config.mdx) is used for static configuration like
-  the cluster name.
-- The [`tctl`](./cli.mdx#tctl) admin tool is used for manipulating dynamic
+- The [configuration file](./config.mdx) is used for static configuration
+  settings such as the cluster name.
+- The [`tctl`](./cli.mdx#tctl) admin tool is used for managing dynamic
```

Comment:
In the CLI doc, it says tctl is "An administrative tool that can configure the Teleport Auth Service."
In this doc, it says "The tctl admin tool is used for managing dynamic records like Teleport users."

Do we want to be consistent in definitions here?  Or is this just contrasting static vs. dynamic configs and can be left as is?

Reply:
Added a suggestion